flink-connector-jdbc-core/src/test/java/org/apache/flink/connector/jdbc/testutils/tables/TableBase.java

```diff
@@ -120,7 +122,7 @@ public int[] getTableTypes() {
                 .toArray();
     }
 
-    public Schema getTableSchema() {
+    public Schema getTableSchema(String pkConstraintName) {
```

Comment:
In the case of Spanner, the contract name of the primary key is different, so it is possible to specify the contract name of the primary key.